src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const discountOfSevenDays = 50;
+  const discountOfThreeOrMoreDays = 20;
+  const total = days * 40;
+
+  if (days >= 7) {
+    return total - discountOfSevenDays;
+  }
 
+  if (days >= 3 || days >= 6) {
```

Comment:
The logical condition `days >= 3 || days >= 6` is incorrect because if `days` is greater than or equal to 6, it is also greater than or equal to 3, making the second part redundant. You should remove `|| days >= 6`.